AISKU/scripts/setActiveCdnVersion.ps1

```diff
@@ -377,10 +377,36 @@ Function ValidateAccess
     }
 }
 
+Function Get-VersionDetails (
```

Comment:
This new function pulls out the "real" version details from versions like
* 2.6.5
* 2.7.0-beta.1
* 2.7.0-beta.201208-001 (This is the nightly build format)